questionary/prompts/press_any_key_to_continue.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any
+from typing import Optional
+
+from prompt_toolkit import PromptSession
+from prompt_toolkit.formatted_text import to_formatted_text
+from prompt_toolkit.key_binding import KeyBindings
+from prompt_toolkit.keys import Keys
+from prompt_toolkit.styles import Style
+from prompt_toolkit.styles import merge_styles
+
+from questionary.constants import DEFAULT_STYLE
+from questionary.question import Question
+
+
+def press_any_key_to_continue(
+    message: Optional[str] = None,
+    style: Optional[Style] = None,
+    **kwargs: Any,
+):
+    """Wait until user presses any key to continue.
+
+    Example:
+        >>> import questionary
+        >>> questionary.press_any_key_to_continue().ask()
+         Press any key to continue...
+        ''
```

Comment:
Is this correct? Or should it be `None`?

Reply:
I think it might be related to the version of `prompt_toolkit`? I distinctly remember that when I initially programmed it, it returned `None`, but when I tried it again recently it returned an empty string `''`.

Reply:
I think it must be None, given that assert result is None is used in the test?

Reply:
This is with prompt_toolkit `3.0.36`.

![Screenshot 2023-05-25 at 12 18 44 PM](https://github.com/tmbo/questionary/assets/14318576/82e6f996-a062-431f-9ecf-7955850bf4b5)

yet, at the same time, the unit test passes. So it seems like the `feed_cli_with_input` does some sort of implicit empty-string-to-none conversion. In real applications, it returns an empty string, we can try and address the unit-test inconsistency in another PR.

Reply:
Ok sounds good!